### DISCUSSION TITLE
Routing allowing only routes with Maximum duration of 11 hours. How to increase this limit so that the solution will have minimal/avoid dropped stores?

### DISCUSSION THREAD

**Discussion by SruthyJohn** [google/or-tools#3381] [Routing (and legacy CP) questions]
Hi Team,
I am getting solution without any dropped stores when trying with below 200 coordinates. But while trying with 500/1000 coordinates i am able to get solution but with around 50% as dropped stores. And the solution is capping at a route maximum duration of 11 hours; whereas i have implicitly given the maximum duration as 86400(24 hours). How will I resolve this so there will be minimal dropped stores.

Please find the gist for the code i tried.
(https://gist.github.com/SruthyJohn/4892461f295a3ad9c386487b8b11cb9e)

**Comment by jmarca** [comment]
On the duration question, you are capping with the depot node operating hours, I believe.

https://gist.github.com/SruthyJohn/4892461f295a3ad9c386487b8b11cb9e#file-program-cs-L8-L9

and 

https://gist.github.com/SruthyJohn/4892461f295a3ad9c386487b8b11cb9e#file-program-cs-L74-L79

**Reply by Mizux** [reply]
It seems it's more a capping due to the Time Windows of all nodes.

Or the vehicle start node TimeWindows to be too late ?
https://gist.github.com/SruthyJohn/4892461f295a3ad9c386487b8b11cb9e#file-program-cs-L87-L92

note: 86400 is the max CumulVar value allowed at each node (easpecially each vehicle end node) aka vehicle will return to depot at 86400 at most.
ref: https://gist.github.com/SruthyJohn/4892461f295a3ad9c386487b8b11cb9e#file-program-cs-L71
But you are also forcing the vehicle start node to be in range [28800, 64800] so at most vehicle will operate in range 28800 -> 86400 aka
86400-28800 == 57600s == 16h.
if you want your vehicle to start in range [28800, 64800] but operate  up to 24h (86400) then your horizon should be 64800+86400.
Beware here you are using a **Time** dimension not a **Duration** dimension (aka a dimension which start at zero in vehicle start node)

**Reply by SruthyJohn** [reply]
Thanks for  your reply. It helped.
